index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hockey Trivia</title>
+    <script type="text/javascript" src="./lib/jquery-3.3.1.min.js"></script>
+    <script type="text/javascript"></script>
+</head>
+
+<body>
+    <h1>Hockey Trivia</h1>
+    <div id="startButton">
+        Start
```

Comment:
In your JS, you are targeting a div which is not calling your start game function.  To add more functionality, I would look into having a button.